core/domain/user_services_test.py

```diff
@@ -1686,31 +1706,30 @@ def test_create_user_contributions_with_bot_user_id_returns_none(self):
     def test_create_user_contributions_already_existing_raises_error(self):
         auth_id = 'someUser'
         user_email = 'user@example.com'
-        created_exp_ids = ['exp1', 'exp2', 'exp3']
-        edited_exp_ids = ['exp2', 'exp3', 'exp4']
-
         user_id = user_services.create_new_user(auth_id, user_email).user_id
+        contributions = user_services.get_user_contributions(user_id)
+        # Check contributions for this user id is created from creating a
+        # New user.
+        self.assertIsNotNone(contributions)
+        # Make sure the variable 'contributions' is actually an object of
```

Comment:
You don't need this, it's already implied by the code below.

In general, use comments to explain *why* a line of code is written (especially in cases where that's unclear), rather than restating what the code is doing.

Reply:
I took this comment out of the file.